nxc/protocols/smb.py

```diff
@@ -665,9 +665,14 @@ def execute(self, payload=None, get_output=False, methods=None):
             except UnicodeDecodeError:
                 self.logger.debug("Decoding error detected, consider running chcp.com at the target, map the result with https://docs.python.org/3/library/codecs.html#standard-encodings")
                 output = output.decode("cp437")
-
-            output = output.strip()
-            self.logger.debug(f"Output: {output}")
+                
+            self.logger.debug(f"Raw Output: {output}")
+            output = "\n".join([ll.rstrip() for ll in output.splitlines() if ll.strip()])
+            self.logger.debug(f"Cleaned Output: {output}")
+            
+            if "This script contains malicious content" in output:
+                self.logger.fail("Script blocked by AMSI")
```

Comment:
Maybe "Execution blocked by AMSI" is more appropriate as this is the normal command execution and not powershell, thoughts?

Reply:
Or maybe "Command execution blocked by AMSI"